lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -814,6 +813,17 @@ def cmd_rename_job(*args)
     true
   end
 
+  #
+  # Tab completion for the rename_job command
+  #
+  # @param str [String] the string currently being typed before tab was hit
+  # @param words [Array<String>] the previously completed words on the command line.  words is always
+  # at least 1 when tab completion has reached this stage since the command itself has been completed
+
```

Comment:
I want to remove this newline so bad, but all the other tab completion methods have it. :/